core/cibox-project-builder/files/drupal7/scripts/tests.yml

```diff
@@ -19,65 +19,27 @@
     update_dependencies: 'yes'
     selenium_jar_path: /var/lib/selenium/selenium.jar
     chrome_driver_path: /var/lib/chrome-driver/chromedriver
+    drupal_core_test_group: 'group,groups,separated,by,comma'
+    run_behat_tests: 'yes'
```

Comment:
You can use `yes` (without quotes) and it'll contain a boolean value. Also, your conditions could be simplified in this way: `when: run_behat_tests` instead of `when: run_behat_tests  == 'yes'`.